src/app/layout.tsx

```diff
@@ -0,0 +1,22 @@
+import type { Metadata } from 'next'
+import { Inter } from 'next/font/google'
+import './globals.css'
+
+const inter = Inter({ subsets: ['latin'] })
```

Comment:
We use [Josefin Sans](https://fonts.google.com/specimen/Josefin+Sans?vfquery=Jos) as Site Font